eval/macro_expension.go

```diff
@@ -9,11 +9,11 @@ import (
 	"grol.io/grol/token"
 )
 
-func DefineMacros(store *object.Environment, program *ast.Statements) {
+func (s *State) DefineMacros(program *ast.Statements) {
```

Comment:
Do you think they have to be exported?

Reply:
yes it's used from repl which isn't in eval